CONTRIBUTING.md

```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
+* `high-priority` - High prioritized issues that needs to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues that are not very important.
+* `in-progress` - Issues that are currently being worked on and assigned.

```

Comment:
Isn't it clear that they are assigned if they are being worked on? Redundant information?